x-pack/qa/security-tools-tests/src/test/java/org/elasticsearch/xpack/security/authc/esnative/tool/ResetPasswordToolTests.java

```diff
@@ -292,8 +292,7 @@ public void testAutoInteractiveModesMutuallyExclusive() throws Exception {
 
     public void testAutoBatchSilent() throws Exception {
         execute(randomFrom("--silent", "-s"), randomFrom("--batch", "-b"), randomFrom("-u", "--username"), user);
-        String output = terminal.getOutput();
-        assertThat(output, hasLength(21)); // password + new line char
```

Comment:
This seems like a place where we do actually care about the newline. Using `lines()` would mask if we were not printing any newline at all. Could we still assert the length?